src/components/shows/heroBanner.js

```diff
@@ -0,0 +1,80 @@
+import React from "react"
+import styles from "./heroBanner.module.css"
+import data from "./data.js"
+import curtains from "../../assets/images/watch/curtains.jpg"
+
+class HeroBanner extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    if (this.props.defined === true) {
+      // If a year HAS been entered...
+      return (
+        <section
+          style={{
```

Comment:
Maybe this would be tidier in the CSS files, setting a class or ID for this section.

Reply:
Done! :)